styles/styles.css

```diff
@@ -38,6 +38,7 @@
   /* Needed? */
   --secondary-grey: #EFEFEF;
   --transparent: #FFFFFFE6;
+  --list-border-grey: #e7e7e7;
```

Comment:
There's now a `--platinum` var for this.

Reply:
fixed